p2p/pre2p/module.go

```diff
@@ -25,22 +25,33 @@ var _ modules.P2PModule = &p2pModule{}
 type p2pModule struct {
 	bus modules.Bus
 
-	listener *net.TCPListener
-	network  typesPre2P.Network
+	listener typesPre2P.TransportLayerConn
```

Comment:
It's meant to be a general purpose transport layer. Not TCP.

AGGREGATE + REJECT>

Reply:
No. Of course. My bad, I gave a bad example by naming it `TCPTransport`. It should be just `Transport` for the reasons I mentioned in the [suggestion](https://github.com/pokt-network/pocket/pull/80#discussion_r895246150) I made before.

I would be open to keeping the name as is, but change it as a part of #86... Since really, per the implementation we have right now, it should keep the name `TransportLayerConn` as the interface defines `Conn`-like behavior like: `Read` and `Write`.

The name should change eventually though, to keep a faithful abstraction, as the `Conn` behavior and the `Transport` behavior will be separated.